src/view/input_parsers/request_data.rs

```diff
@@ -62,9 +63,68 @@ mod parsers_request_items {
     use serde_json::Map;
 
     use super::*;
-    use crate::app::services::request::entities::requests::BodyPayload;
     use crate::utils::regexes;
 
+    // Github Issue #8
+    // Parse "<key>:=<null>"
+    // Parse "<key>:=[-]<number>"
+    // Parse "<key>:=true|false"
+    // Parse "<key>:=[<null>, <string>, <number>, <boolean>, <array>, <object>]"
+    // Parse "<key>:={"<key>": <null>|<string|number|boolean|array|object>"}"
+    pub fn non_string_body_value(
```

Comment:
Add more unit tests for this function to validate following cases:
- key:={}
- key:=[]
- key:="{}"
- key:="false"
- key:={"name":"Talis"}
- key:=[1,2,3]
- key:=[null,{},["a"]]

Also make quick tests to assert if these groups of inputs produces same output
group 1: 
- key:="[]"
- key:=[]
- key:='[]'

group 2:
- key:="false"
- key:=false
- key:='false'

group 3:
- key:="null"
- key:=null
- key:='null'

If you wish you can do the same for numbers and objects